tests/unit/test_projection_engine_integration.py

```
@@ -0,0 +1,94 @@
+    event_repo = EventRepository(async_session)
+    registry = ProjectionRegistry()
+
+    # Register a dummy projection handler.
+    async def handler(state: dict, event):
+        state["count"] = state.get("count", 0) + 1
+        return state
+
+    registry.register("dummy", 1, handler)
+
+    # Create definition row.
+    defn = await repo.create(
```

Flake8 (7.3.0)

[error] local variable 'defn' is assigned to but never used

(F841)